### DISCUSSION TITLE
Another twistd server is Running ,PID 1

### DISCUSSION THREAD

**Discussion by PEQSPC** [containers/podman#21167] [General]
![2024-01-05 09_53_53-Podman containers - admin@globaleaks — Mozilla Firefox](https://github.com/containers/podman/assets/124091719/3bc7299b-aac8-41f1-a5e8-cfede4c56f42)
Does anyone know how to solve this problem?
I'm using alma linux 9.3.
podman version 4.6.1

**Comment by rhatdan** [comment]
Looks like  you have two twistd servers running attempting to connect to the same port.  Perhaps the first and second container are binding to the same port?

Showing us the podman command used to launch the container would help.

**Reply by PEQSPC** [reply]
/usr/bin/python3 /usr/bin/globaleaks --working-path=/var/globaleaks/ -n
![imagem](https://github.com/containers/podman/assets/124091719/393aed32-ba01-481d-bc2e-b3f813f939d0)

**Comment by marchrius** [comment]
Hi, I'm facing the same problem. In my case, in a docker container, the __globaleaks.pid__ file located in the working path was preventing execution. I had to remove it manually to restore the execution.
